hassio_aw2m/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ghcr.io/neilenns/ambientweather2mqtt:latest
+
+USER root
+RUN apk add --update jq
+
+COPY startup.sh /home/node/app/startup.sh
+RUN chmod a+x /home/node/app/startup.sh
+
+#ENTRYPOINT [ "ls" ]
```

Comment:
This probably came from the sample but since it isn't needed it can be removed instead of commented out.